src/components/structures/HomePage.js

```diff
@@ -91,11 +91,15 @@ class HomePage extends React.Component {
         this._unmounted = true;
     }
 
-    onLoginClick() {
+    onLoginClick(ev) {
+        ev.preventDefault();
+        ev.stopPropagation();
```

Comment:
Is `preventDefault` enough by itself, or do you really need `stopPropagation` also?

Reply:
`preventDefault` is enough for our use case, although we manage the `onClick` handling and should prevent propagation as well to be safe.